contrib/ansible/inventory

```diff
@@ -1,10 +1,9 @@
 
 [masters]
-kube-master-test-01.example.com
+kube-master-test-[1:3].example.com
 
-[etcd]
-kube-master-test-01.example.com
+[etcd:children]
+masters
```

Comment:
@eparis maybe we so another example file here like we did with the openstack config file. Have this file become a legitimate example file with documentation surrounding it.